tests/Equinox.EventStore.Integration/Infrastructure.fs

```diff
@@ -14,12 +14,15 @@ type FsCheckGenerators =
 
 #if STORE_POSTGRES || STORE_MSSQL || STORE_MYSQL
 open Equinox.SqlStreamStore
-#else
-#if !STORE_EVENTSTORE_LEGACY
+#endif
```

Comment:
not sure if it's an improvement but I find reading a series of

```fsharp
#if
// ....
#endif
```

easier than if-else-if because it doesn't end with `n` `#endif`s

Happy to revert if you prefer the elses

Reply:
I do wish there was a #elif but yeah I guess this is for the best